src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,9 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo comment={comment} />
```

Comment:
Each `CommentInfo` component in the list should have a unique `key` prop. Consider using a unique identifier from the `comment` object, such as `comment.id`, to set the `key` prop.